zuul.d/jobs.yaml

```diff
@@ -645,7 +645,7 @@
       Run Kubernetes E2E Conformance tests against release-1.14 branch of Kubernetes
     nodeset: ubuntu-xenial-citynetwork
     vars:
-      go_version: '1.12.1'
```

Comment:
This doesnt need to change , release-1.14 supports 1.12.1

Reply:
It's a minor version, besides 1.12.10 contains security fixes: https://golang.org/doc/devel/release.html#go1.12